### ISSUE TITLE
Selecting a node which is covered by another

### ISSUE THREAD

**Issue by AndyK70** [PromGames/Coursedrawer#16]
When nodes are close together they can't really selected by mouse.

Provide two shortcuts (keys) to select previous node and next node.
Then you can select whichever node you can (of that course) and brachiate to the desired node.

**Comment by PromGames** [comment]
Yes, that is very important. At least if we can move to previous or next node with the up and down key, or W-S if up and down can not be used, and also the information of the selected WP has to be refreshed in the Way Point panel Info.

**Comment by jmj486** [comment]
It would be nice if W-S was for Next-prev. and arrow keys could be used to move the nodes
